architecture/core_concepts/routes.adoc

```diff
@@ -1183,3 +1185,69 @@ $ oc expose service/<name> --hostname="m.api.openshift.org"
 $ oc expose service/<name> --hostname="kates.net"
 $ oc expose service/<name> --hostname="api.kates.net"
 ----
+
+
+[[disable-namespace-ownership-check]]
+== Disabling the Namespace Ownership Check
+
+By default, different paths to a host name (and subdomains for wildcard routes)
+can only be claimed by routes in the same namespace as the owning route.
+
+For example, if namespace `ns1` creates the oldest route r1 with host `*_www.abc.xyz_*`, then it
+owns that host name and subdomain (`abc.xyz`) for wildcard routes.  If another
```

Comment:
The ownership/claim space needs to cover all that is there. I want the user to understand the boundaries of the space. There may be a better place or way to do this. I will think about it.